docs/edex/new-grid.md

```diff
@@ -15,105 +17,52 @@ If the parameters in the grib file haven't been previously specified, another ch
 
 ## Ingest an Unsupported Grid
 
-### Grib Products
+### Download Test Data
 
-1. Download an example grib1 file and rename to a `*.grib` extension, then copy to the manual ingest point `/awips2/data_store/ingest/` 
+Download an example grib2 file (make sure the extension is `.grib2` or EDEX may not recognize it), and then copy to the manual ingest point `/awips2/data_store/ingest/` 
```

Comment:
"(make sure the extension is `.grib2` or EDEX may not recognize it)"
and 
"Remember that the data distribution file..."  (line 28)

is pretty repetitive, I suggest combining those two sentences